index.html

```diff
@@ -29,352 +32,374 @@
+                ><img src="images/vectorm.png" alt="mixx icon link on mobile"
+              /></a>
+            </li>
+            <li>
+              <a href=""
+                ><img
+                  src="images/vectorhead.png"
+                  alt="github icon link on mobile"
+              /></a>
+            </li>
+            <li>
+              <a href=""
+                ><img
+                  src="images/vectorhand.png"
+                  alt="click the join link below"
+              /></a>
+            </li>
+          </ul>
         </div>
       </div>
-      <div id="section-2">
-        <!---start markup for s2 Desktop-->
-        <div class = "s2-container-dt">
-         
-              <div class = "card-text-2">
-          <h2>Multi-Post-Stories</h2>
-          <div class = "list-container2-dt">
-            <ul>
-              <li><a href="" class="text">FACEBOOK</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-2">Full Stack Dev</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-3">2015</a></li>
-            </ul>
-          </div>
-          <div>
-          <p>
-            Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
-          </p>
-          </div>
-          <div class = "base-links">
-            <ul>
-              <li><a href="" class="base-link-text1">html</a></li>
-              <li><a href = "" class = "base-link-text2">Ruby.on.rails</a></li>
-              <li><a href="" class="base-link-text2">css</a></li>
-              <li><a href="" class="base-link-text3">javascript</a></li>
-            </ul>
-          </div>
-          <div class = "base-button">
-          <a href = "#">See Project</a>
-          </div>
-          </div>
-          <div class = "img-container2-dt">
-            <img src = "images/desktop-section-2.png" alt = "The book"/>  
+      <section id="works">
+        <div id="section-1">
+          <!----Markup for desktop card-1-->
+          <div class="s1-container-dt">
+            <div class="img-container">
+              <img src="images/desktop-section-1.png" alt="" />
             </div>
-  
-  </div>
-        <!---End Mark up for s2 Desktop -->
-        
-    <!------End of styling for Desktop card 2-->
-        <div class="s2-container">
-          <div class="image-container2"></div>
+            <div class="card-text">
+              <h2>Tonic</h2>
+              <div class="list-container">
+                <ul>
+                  <li><a href="" class="text">CANOPY</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
+                  <li><a href="" class="text-2">Back End Dev</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
+                  <li><a href="" class="text-3">2015</a></li>
```

Comment:
- [ ] I noticed here the alt is empty,, kindly add a text describe the image for accessibility issues 🙏🏻